lib/atomic/misc.less

```diff
@@ -149,14 +149,14 @@
     // Borrowed from Wordpress
     // https://make.wordpress.org/accessibility/2015/02/09/hiding-text-for-screen-readers-with-wordpress-core/
     border: 0;
-    clip: rect(1px, 1px, 1px, 1px);
+    clip: rect(var(--su-static1), var(--su-static1), var(--su-static1), var(--su-static1)); // rect(1px, 1px, 1px, 1px)
```

Comment:
I see sometimes you add comments sometimes you don't. Is there a rule you use to decide that (e.g. if it is just a `var` is self-explanatory enough)?